components/common/GridItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export default function GridItem(props) {
+  const isExternalLink = props.link && /^https?/.test(props.link);
+  const linkAttributes = {
+    href: props.link,
+    rel: 'noopener',
```

Comment:
`noopener` is sufficient to block the security threat but it seems that older browser don't support it. Thanks, I'll update the code!

[Source](https://mathiasbynens.github.io/rel-noopener/)